### ISSUE TITLE
[EN][Quest][Loot] Non-quest item drop, if quest objective.

### ISSUE THREAD

**Issue by wtRCH** [Atlantiss/NetherwingBugtracker#3432]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: When you complete quest objective for Stocking Jetsteam (getting 4 Chunk of Boar Meat) and accidentaly cook it, you no longer can loot the Chunk of Boar Meat from the mobs as long as you have the same quest in your quest log.

`I think It is global issue.`

**Current behaviour**: Can't loot more Chunk of Boar Meat after completing quest objective, regarding it not being quest item but normal item.

**Expected behaviour**: Chunk of Boar Meat should drop regardless of quest objective.

**Server Revision**: 2431

**Comment by Tenoutoften** [comment]
https://github.com/Atlantiss/NetherwingBugtracker/issues/1689

**Comment by wtRCH** [comment]
@Tenoutoften 
It's more about the normal items you can drop without a quest, that doesn't drop after you complete the quest objective they are part of.

**Comment by Wolffenstein** [comment]
That is not true for Chunk of Boar Meat. It drops fine for me even after completing the quest (remember it is not a guaranteed drop).
